src/engine/input/key_mapping/InputEventItem.cs

```diff
@@ -401,6 +401,13 @@ public override void _Input(InputEvent @event)
         OnKeybindingSuccessfullyChanged();
     }
 
+    public void FinishRebind(InputEvent inputEvent)
```

Comment:
I'm wondering if it would make sense for the above method (around line 387 to call this single helper method) to call this instead of duplicating the event assign and calling `OnKeybindingSuccessfullyChanged`